api/tests/test_asset_log.py

```diff
@@ -337,6 +338,109 @@ def test_authenticated_admin_user_get_of_asset_logs_invalid_filter(
         self.assertEqual(response.status_code, 200)
         self.assertEqual(len(response.data["results"]), 0)
 
+    @patch("api.authentication.auth.verify_id_token")
+    def test_filter_asset_logs_by_year(self, mock_verify_id_token):
+        mock_verify_id_token.return_value = {"email": self.admin_user.email}
+        AssetLog.objects.create(
+            checked_by=self.security_user, asset=self.test_other_asset, log_type=CHECKIN
+        )
+        asset_logs_url = (
+            f"{self.asset_logs_url}?year={self.test_other_asset.created_at.year}"
```

Comment:
I believe this filter is for the year asset-logs were created and not when the assets they log were created.
You can update the `created_at` attribute of the created log to set a custom year for your assertion. Check out [this](https://docs.djangoproject.com/en/2.2/ref/models/querysets/#django.db.models.query.QuerySet.update).